src/main/java/com/launchacademy/giantleap/models/FashionItemReview.java

```diff
@@ -38,11 +39,27 @@
     @Column(name="comment")
     private String comment;
 
-    @URL
     @Column
     private String photo;
 
     @Column
     private Integer rating;
 
+<<<<<<< HEAD
```

Comment:
You still have the remnants of a merge conflict here. 
You need to ensure you remove the <<<<<<< HEAD, the =====, and the >>>>>> (and hash)